CSharp_Course_Project/Task6_LongestString.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+
+namespace CSharp_Course_Project
+{
+    /// <summary>
+    /// Concatenated string with unique chars
+    /// </summary>
+    /// <owner> Kirill Novikov </owner>
+    class Task6_LongestString
+    {
+        public static void LongestString()
+        {
+            string firstString = Task6_LongestString.GetRandomString();
```

Comment:
Please add empty lines between of logical parts of code.

Reply:
fixed